### DISCUSSION TITLE
有关自动肉鸽

### DISCUSSION THREAD

**Discussion by z2oop** [MaaAssistantArknights/MaaAssistantArknights#8221] [Ideas]
我想这长期探索的图标更改你肯定吃惊，这也带来了问题，无法进行自动肉鸽！且无法移动至导航！
![image](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/97105028/8aaf149e-f0bf-4b5e-88c1-48c1453e915a)
![image](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/97105028/aa46fb96-fe25-4a36-b5cb-a8f659a41d63)
如果可以的话请修一下这个“BUG”，这毕竟影响了正常挂机。
![image](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/97105028/b932abfd-0dab-47b8-999c-7da263884f33)
![image](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/97105028/656a378e-ea94-44dc-98da-96bf02a67b27)
“这应该不会耗费你们太长时间，对吧？”

**Comment by z2oop** [comment]
开启MAA后我挂机了一下午，都卡在这，debug/roguelike里的照片都300MB了

**Comment by SherkeyXD** [comment]
#8248
